kahuna/public/js/edits/image-editor.js

```diff
@@ -5,13 +5,17 @@ import './service';
 import '../image/service';
 import '../usage-rights/usage-rights-editor';
 import '../components/gr-archiver-status/gr-archiver-status';
+import {collectionsApi} from '../services/api/collections-api';
+import {rememberScrollTop} from '../directives/gr-remember-scroll-top';
 
 
 export var imageEditor = angular.module('kahuna.edits.imageEditor', [
     'kahuna.edits.service',
     'gr.image.service',
     'kahuna.edits.usageRightsEditor',
-    'gr.archiverStatus'
+    'gr.archiverStatus',
+    collectionsApi.name,
```

Comment:
I would argue moving towards this way as it's stops having funny-little-strings-that-can-change-at-any-time lying all over the place.

Strings are for sentences.